### PR TITLE
[Core] Login Page UI

### PR DESCRIPTION
**Pull request by darshkpatel**
* Basic Redirection Logic
* Add grommet globally in project
* Added UI for Login Page

### CODE REVIEW

**Comment by darshkpatel** [comment]
Ref: #4

**Comment by anshulrgoyal** [comment]
@darshkpatel instead of the master you can merge it in the #9

**Comment by darshkpatel** [comment]
> @darshkpatel instead of the master you can merge it in the #9

Changed the base

File: .vercel/project.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"projectId":"Qmf3ydjvNQWmSBkPurHi2GGgMbn7oQtwgZGSHJjcp4XpSa","orgId":"JAQ78UA8gBjnVXgKVRupyDO5"}
```
Comment (chuabingquan) [inline .vercel/project.json:1]: Should these IDs be exposed on GitHub? If not could you look into substituting them with environment variables instead?

Comment (darshkpatel) [reply .vercel/project.json:1]: My bad, forgot to add gitignore entry for vercel , i'll create a new project to invalidate the exposed IDs

Comment (darshkpatel) [reply .vercel/project.json]: @chuabingquan removed in e929d0b and added to gitignore in e2d47d5

Comment (chuabingquan) [reply .vercel/project.json]: Looks good to me.

**Pull request merged**